centreon/src/Centreon/Domain/Log/LoggerTrait.php

```diff
@@ -222,15 +238,29 @@ private function log($level, string $message, array $context = [], callable $cal
 
     /**
      * @param string $message
+     *
      * @return string
      */
     private function prefixMessage(string $message): string
     {
         $debugTrace = debug_backtrace();
```

Comment:
This method uses only `file` & `line` from the trace.
It is a perf increase + safety to avoid retrieving arguments with `DEBUG_BACKTRACE_IGNORE_ARGS`.
> php reference manual for [debug_backtrace()](https://www.php.net/manual/en/function.debug-backtrace.php)

It can be usefull also to set a limit, high limit, but a limit in order to avoid "_infinity_" ∞ (second optional argument).
```suggestion
        $debugTrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```